app/blog/20161005-frontend-build-instructions

```diff
@@ -0,0 +1,80 @@
+---
+title: Welcome to the n.io Front-End Team!
+author:  Trisha Allin
+posted: 20161005
+summary:
+  Everything you need to get your front-end build up and running.
+_options:
+  layout: app/layouts/blog.static.hbs
+---
+
+
+## Install your favorite shell script and text editor!
+Feel free to use whichever terminal shell and text editor you work with best.
+
+Recommendations:
+1. Shell script: zsh/oh-my-zsh
+This shell allows you to theme, has helpful plug-ins and shows you which branch you are on.
+https://github.com/robbyrussell/oh-my-zsh
+2. Text editor: Sublime Text, Atom
+Patrick will probably supply you with sublime… but here’s the link just in case
+https://www.sublimetext.com/
+
+
+
+## Let’s Install Homebrew!
+http://brew.sh/index.html
+
+Paste this into your terminal:
+
+`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+
+## Let’s Install Node.js!
+https://nodejs.org/en/
+
+Now that you have Homebrew all you have to do is type in the terminal:
+
+`brew install node`
+
+## Let’s Get Synced with GitHub!
+###Generate your ssh key
+https://help.github.com/articles/generating-an-ssh-key/
+1. See if you already have an ssh key by typing in terminal: ls -al ~/.ssh
```

Comment:
I would put the "ls -al ~/.ssh" in tick marks just like the other code samples so it stands out in the post!